config/clusters/uwhackweeks.cluster.yaml

```diff
@@ -0,0 +1,174 @@
+name: uwhackweeks
+provider: aws
+aws:
+  key: secrets/uwhackweeks.json
+  clusterType: eks
+  clusterName: uwhackweeks
+  region: us-west-2
+support:
+  config:
+    prometheus:
+      server:
+        resources:
+          requests:
+            cpu: 1
+            memory: 4Gi
+          limits:
+            cpu: 4
+            memory: 8Gi
+    cluster-autoscaler:
+      enabled: true
+      autoDiscovery:
+        clusterName: uwhackweeks
+      awsRegion: us-west-2
+    grafana:
+      ingress:
+        hosts:
+          - grafana.uwhackweeks.2i2c.cloud
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - grafana.uwhackweeks.2i2c.cloud
+hubs:
+  - name: staging
+    domain: staging.uwhackweeks.2i2c.cloud
+    helm_chart: daskhub
+    auth0:
+      connection: github
+    config: &uwhackweeksConfig
+      scratchBucket:
+        enabled: false
+      basehub:
+        nfs:
+          pv:
+            # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+            mountOptions:
+              - rsize=1048576
+              - wsize=1048576
+              - timeo=600
+              - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+              - retrans=2
+              - noresvport
+            serverIP: fs-0b36aa23f1d92e4da.efs.us-west-2.amazonaws.com
+            baseShareName: /
+        jupyterhub:
+          custom:
+            homepage:
+              templateVars:
+                org:
+                  name: ICESat Hackweek
+                  logo_url: https://icesat-2hackweek.github.io/assets/images/ICESat2.png
+                  url: https://icesat-2hackweek.github.io
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name:
+                  url: https://www.openscapes.org/
```

Comment:
Funded by openscapes? I guess it is a leftover from copy-pasting the yaml file? cc @yuvipanda